src/fontra/views/editor/editor.js

```diff
@@ -852,11 +854,27 @@ export class EditorController {
   }
 
   canCopy() {
-    return true;
+    return this.sceneController.selection.size > 0;
```

Comment:
I may want to revisit this: I want an empty selection to mean "copy the whole glyph"
Perhaps I also want copy to work if a glyph is merely only selected, and not only in edit mode.

Reply:
We can use selectedGlyph if that makes more sense?

Reply:
Yeah, I think so. Funnily enough, the behavior I just described already works :)